databus-api/src/main/java/com/bazaarvoice/emodb/databus/api/DefaultSubscription.java

```diff
@@ -14,7 +14,7 @@
 public final class DefaultSubscription implements Subscription {
     // Currently, by default, subscription event ttl limit is set to 365 days,
     // but that could be changed in future
-    private final static Duration EVENT_TTL_LIMIT = Duration.ofDays(365);
+    private final static Duration TTL_LIMIT = Duration.ofDays(365);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)

Reply:
fixed